CHANGELOG.rst

```diff
@@ -35,6 +35,8 @@ Improved Documentation
   xdist could identify the wrong test as failing when test crashes at teardown.
   To address this issue, xdist now marks test as complete at teardown.
 
+- 168.trivial: add long option `--numprocesses` as alternative for `-n`.
```

Comment:
The template is clearer now thanks. For people like me who apparently can't read comments in CHANGELOG files you might want to add a line **Do not edit  CHANGELOG.rst** to the template.